BlakeDonn/rancid-tomatillos
# Architecture

### What's this PR do?  

- Add initial structure for the functional and class components
- Add very basic css to provide initial overview
- Add initial API get call and handle

### Where should the reviewer start?  


- View the component data flow and see where things could be adjusted
- View the mount functionality and make sure it is working correctly

### How should this be manually tested?  

- Start by opening the react-app and viewing the movie layout 
- Open react dev tools and check out component layout

### Any background context you want to provide?  

- Still debating how our flow should work in components (what should be classes and what shouldnt)

### What are the relevant tickets?  

closes #31 closes #21 closes #20 
Progresses: #2 #4 #28 #29
